src/components/pages/lessons/LessonInfo.tsx

```diff
@@ -84,10 +84,9 @@ const CopyToClipboard: FunctionComponent<{
   stringToCopy: string
   className?: string
 }> = ({stringToCopy, className = ''}) => {
+  const [copied, setCopied] = useState(false)
+  const [state, copyToClipboard] = useCopyToClipboard()
   if (stringToCopy) {
-    const [copied, setCopied] = useState(false)
```

Comment:
@DrShpongle no hooks can run inside an `if` or any other conditional. This is tricky, but now the linter should break if it happens.

Reply:
Didn't know it, thanks!